### DISCUSSION TITLE
Connect to chain not in default chain

### DISCUSSION THREAD

**Discussion by stoplion** [wevm/wagmi#146] [Q&A] [answered]
Hi, Is there is a way to connect to a chain not provided in defaultChains? I.E. Fantom.

**Comment by tmm** [comment, marked as answer]
Yup! For all the built-in connectors, you can pass in whatever chains you want. An example with the [injected connector](https://wagmi-xyz.vercel.app/docs/connectors/injected):

```ts
import { chain, defaultChains, defaultL2Chains } from 'wagmi'
import { InjectedConnector } from 'wagmi/connectors/injected'

const connector = new InjectedConnector({
  chains: [...defaultChains, ...defaultL2Chains, chain.hardhat],
})
```

You can also create custom chains that conform to the `Chain` type:

```ts
import { Chain } from 'wagmi'

const fantomOpera = {
  id: 250,
  name: 'Fantom Opera',
  nativeCurrency: { name: 'Fantom', symbol: 'FTM', decimals: 18 },
  rpcUrls: ['https://rpc.ftm.tools'],
  blockExplorers: [
    {
      name: 'FTMScan',
      url: 'https://ftmscan.com',
    },
  ],
}

const connector = new InjectedConnector({
  chains: [fantomOpera],
})
```

Feel free open a PR to add Fantom to wagmi.

**Reply by JanRuettinger** [reply]
I tried to to do the same with a different chain. It works fine I can connect however `useBalance` still returns the ETH balance. Why is that? Could you extend the example so that the `useBalance` also works?
